pmultiqc/modules/quantms/quantms.py

```
@@ -1013,6 +1135,61 @@ def draw_search_engine(self):
+            # MGF
+            if self.mgf_paths:
+                psm = pd.merge(psm,
+                               self.mgf_rtinseconds[["spectrumID", "filename", "retention_time"]],
+                               on=["filename", "spectrumID"], how="left")
+            # mzml
+            # elif self.ms_paths:
+            #     print("mzid_CalHeatMapScore(): need mzml from ms_paths!")
+        
+        for name, group in psm.groupby('stand_spectra_ref'):
+            sc = group['missed_cleavages'].value_counts()
+            mis_0 = sc[0] if 0 in sc else 0
```

Ruff

Use `sc.get(0, 0)` instead of an `if` block

Replace with `sc.get(0, 0)`

(SIM401)